package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
+    "build": "next build && cp ./CHANGELOG.md ./public/changelog.md",
```

Comment:
This works because it runs on _build_ time. I wouldn't change how this script works, but this leads to my question, why do you need to commit `public/CHANGELOG.MD` to the repo?

I don't think it's necessary to commit this file to git.

`npm run builds` runs automatically in vercel and in docker when the app builds, so the changelog file will be copied over correctly. If you want to keep it committed in the repo, then you'd have to run `npm run build` before making PRs to keep both changelog files synced and typically you don't run `npm run build` locally.

If you do want/need it in git that's okay but just curious why it's there.

Reply:
The file is there because, in the past (with ruby environments), I've ran into issues copying contents to files if the file didn't already exist. I kept it in as a precaution. 

Is the concern that the` /public/CHANGELOG.MD` file in the _repo_ won't reflect the changes that are live in vercel and aws because the changes created when running `npm run build` or `docker build` would only update the  file in the container?

Reply:
Yes, that's right. Both files can be out of sync in the repo and in vercel (or aws) if you _only_ update the main changelog file at the root of the repo. You and Emma (or anyone else, really) would have to remember to run `npm run build` before committing and pushing code so that they're synced. It's not common to have to run that command before pushing code.

Try this in vercel and aws without committing the `public/CHANGELOG.md` file and let me know what do you think.